core/runner.py

```diff
@@ -320,8 +349,12 @@ def _average(self, job):
         """
         Average the weights in the job weighted by their omegas.
         """
-        deserialized_new_weights = deserialize_weights(job.new_weights)
-        averaged_weights = self._weighted_running_avg(job.weights, deserialized_new_weights, job.sigma_omega, job.omega)
+        # TODO: Figure out how to assert that this job actually has weights/new_weights.
+        # assert isinstance(job.weights, np.ndarray), "No weights supplied to average!"
+        # assert isinstance(job.new_weights, np.ndarray), "No new_weights supplied to average!"
```

Comment:
Could you briefly explain the TODO?

Reply:
Basically I want
`assert job.new_weights`
But that gives me 'truth value of array is ambiguous'

Reply:
You're basically checking that it's not empty right? Why not convert to `list` and then assert?